### DISCUSSION TITLE
Windows 11 Pro 32 bit Error Message

### DISCUSSION THREAD

**Discussion by thumperprice** [leecher1337/ntvdmx64#222] [Q&A]
Hello,

I just tried installing NTVDMx64 on my 64 bit system but after running install.bat, I get a message saying I'm trying to install on a 32 bit system. I'm definetly 64 bit Windows 11 Pro, MSI B550 Tomahawk MB, Ryzen 5 4600G, 64GB Memory, etc.... Anybody else run into this?

Thanks,
Donald

**Comment by leecher1337** [comment]
Can you please paste the output of:

```
reg query HKLM\Hardware\Description\System\CentralProcessor\0
```
? I wonder why there is an "x86" in it which causes the detection to think that you are on 32bit.

If you want to force install, just remove the following paragraph from install.bat:
```
echo ---------------------------------------------
echo Checking machine, please wait...
echo ---------------------------------------------
reg query HKLM\Hardware\Description\System\CentralProcessor\0 | Find /i "x86" >nul
if not errorlevel 1 (
  echo You appear to be running this installation on a 32bit machine.
  echo This NTVDMx64 is only meant to be used on an x64 machine, please use 
  echo NTVDM shipped with your windows installation instead.
  echo Installation aborted
  pause
  goto fini
)
```

**Reply by thumperprice** [reply]
Here is a screen capture of the output.
![Computer Readout](https://user-images.githubusercontent.com/129441206/228968458-06a1a9ee-117f-4f0e-a8eb-11bbf5d2c8d1.jpg)

**Comment by leecher1337** [comment]
Thanks, fixed
